packages/core/src/components/Tooltip/__stories__/Tooltip.stories.tsx

```diff
@@ -85,31 +110,49 @@ export const TooltipWithImage = () => (
 
 export const Positions = {
   render: () => {
+    // The modifier's purpose is to prevent the tooltip from being displayed when the user scrolls the story upwards / downwards.
+    // Therefore, there is no need to move this prop in your implementations.
+    const modifiers = [
+      {
+        name: "preventOverflow",
+        options: {
+          mainAxis: false
+        }
+      },
+      {
+        name: "flip",
+        options: {
+          // @ts-ignore
+          fallbackPlacements: []
+        }
+      }
+    ];
+
     return (
-      <div className="monday-storybook-tooltip_positions">
-        <div>
+      <Flex gap={16}>
+        <div style={{ padding: "0 64px 68px 0", margin: "0 32px" }}>
           {/* The modifier's purpose is to prevent the tipseen from being displayed when the user scrolls the story upwards / downwards.
          Therefore, there is no need to move this prop in your implementations. */}
           <Tooltip modifiers={modifiers} content="Top" shouldShowOnMount position="bottom" open>
             <div />
           </Tooltip>
         </div>
-        <div>
+        <div style={{ padding: "50px 0 0 0", margin: "0 32px" }}>
           <Tooltip modifiers={modifiers} content="Bottom" shouldShowOnMount open>
             <div />
           </Tooltip>
         </div>
-        <div>
+        <div style={{ padding: "0 32px 8px 32px", margin: "0 32px" }}>
           <Tooltip modifiers={modifiers} content="Left" position="right" shouldShowOnMount open>
             <div />
           </Tooltip>
         </div>
-        <div>
+        <div style={{ padding: "0 64px 8px 64px", margin: "0 64px" }}>
```

Comment:
I think we should think of a simpler layout to prevent such cases (not right now)